lib/usecases/run/GetAllRunsUseCase.js

```diff
@@ -233,6 +235,15 @@ class GetAllRunsUseCase {
                 });
             }
 
+            if (dataPassIds) {
+                const runNumbers = (await RunRepository.findAll(dataSource.createQueryBuilder()
+                    .set('attributes', ['runNumber'])
+                    .set('raw', true)
+                    .setModel('Run')
+                    .whereAssociation('dataPass', 'id').oneOf(...dataPassIds))).map(({ runNumber }) => runNumber);
```

Comment:
TBH here, you should simply use raw query object, as it is done for tags, don't bother with query builder

Reply:
applied